coils_tet_mesh.py

```diff
@@ -0,0 +1,189 @@
+import pandas as pd
+import numpy as np
+import subprocess as sbp
+import cubit
+
+
+def coils_tet_mesh(coils_path, total_coil_count, log_general_path,
+                   exo_general_path, h5m_general_path):
+    """Create a tet mesh of the magnet coils using Cubit 
+    and export them as .exo and .h5m files
+
+        Arguments:
+            coils_path (str): absolute path to the coils .step file
+                containing all of the coil geometries to be meshed.
+            total_coil_count (int): number of coils contained in the .step file
+            log_general_path (str): absolute path to a .log file that will be
+                created by Cubit. 
+                **Note: Do not include any numerical index at the end of
+                the file name because the function iterates over every coil in the .step
+                file and will append the index to the .log filename.**
+            exo_general_path (str): absolute path to a .exo file that will be
+                created by the function. 
+                **Note: Do not include any numerical index at the
+                end of the file name because the function iterates over every coil in
+                the .step file and will append the index to the .exo filename.**
+            h5m_general_path (str): absolute path to a .h5m file that will be
+                created by the function. 
+                **Note: Do not include any numerical index at the
+                end of the file name because the function iterates over every coil in
+                the .step file and will append the index to the .h5m filename.**
+        """
+
+    # GEOMETRY AND MESH CREATION
+
+    # Start Cubit - this step is key
+    cubit.init(['cubit', '-nojournal'])
+
+    # Import coils
+    cubit.cmd(f'import step "{coils_path}" heal')
+    cubit.cmd('volume  size auto factor 5')
+
+    # For-loop iterates over all 48 coils included in the coils.step file
+    for coil_num in np.arange(1,total_coil_count):
+        
+        # Create the mesh
+        cubit.cmd(f'volume {coil_num} scheme tetmesh')
+        cubit.cmd(f'volume {coil_num} scheme tetmesh')
+        cubit.cmd(f'mesh volume {coil_num}')
+
+        # Set log file path and initiate logging
+        log_split = log_general_path.split('.')
```

Comment:
Do this once before the loop, perhaps name the components for clarity something like `log_prefix` and `log_ext`?